### PR TITLE
[sc-3023] Display page indicator when available

### PR DESCRIPTION
**Pull request by mpellerin42**
- Display page indicator on PDF tile when page number is available
  - When clicking on a paragraph, if not found by pdfjs, display the corresponding page

### CODE REVIEW

File: libs/search-widget/src/tiles/pdf-tile/PdfTile.svelte [hunk 1]
```diff
@@ -56,17 +55,26 @@
...
+          }));
+          return paragraphList;
+        }),
+      ),
     viewerStore.results,
     isSearchingInPdf,
   ]).pipe(
     map(([globalResult, inPdfResults, isInPdf]) => (isInPdf ? inPdfResults : globalResult)),
     map((results) => results || []),
+    tap((results) => console.log(results)),
```
Comment (ebrehault) [inline libs/search-widget/src/tiles/pdf-tile/PdfTile.svelte:77]: to be cleaned up

File: libs/search-widget/src/tiles/pdf-tile/PdfTile.svelte [hunk 2]
```diff
@@ -101,12 +109,14 @@
     if (resultIndex > 0) {
       resultIndex -= 1;
       selectedParagraph = paragraphList[resultIndex];
+      console.log(`openPrevious - selectedParagraph`, selectedParagraph);
```
Comment (ebrehault) [inline libs/search-widget/src/tiles/pdf-tile/PdfTile.svelte:112]: clean up

File: libs/search-widget/src/tiles/pdf-tile/PdfTile.svelte [hunk 3]
```diff
@@ -101,12 +109,14 @@
...
     }
   };
   const openNext = () => {
     if (resultIndex < paragraphList.length - 1) {
       resultIndex += 1;
       selectedParagraph = paragraphList[resultIndex];
+      console.log(`openNext - selectedParagraph`, selectedParagraph);
```
Comment (ebrehault) [inline libs/search-widget/src/tiles/pdf-tile/PdfTile.svelte:119]: clean up

File: libs/search-widget/src/tiles/pdf-tile/PdfViewer.svelte [hunk 4]
```diff
@@ -26,13 +27,38 @@
...
   $: pdfViewer && !paragraph && unselectText();
   $: containerOffsetWidth && resize();
 
   onMount(() => {
     loadPdf();
+
+    subscriptions.push(
+      updateTextLayerMatch$
+        .pipe(
+          debounceTime(100),
```
Comment (ebrehault) [inline libs/search-widget/src/tiles/pdf-tile/PdfViewer.svelte:47]: maybe a comment to explain the debounce

**Review by ebrehault — APPROVED** [review]

**Review by operramon — APPROVED** [review]

**Pull request merged**